rich/traceback.py

```diff
@@ -544,6 +572,18 @@ def __rich_console__(
                     (f"{stack.exc_type}: ", "traceback.exc_type"),
                     highlighter(stack.syntax_error.msg),
                 )
+            elif stack.grouped_traces:
+                group_exception_renderable = Panel(
+                    Group(*[Group(*self._render_trace(console=console, options=options, trace=trace, child_index=index + 1)) for index, trace in enumerate(stack.grouped_traces)]),
+                    title=f"[exception_group.title]{stack.exc_type} [dim]{highlighter(stack.exc_value)}",
```

Comment:
I'd love to see support for PEP-678 `__notes__` here (and in the next `elif` clause, for non-grouped exceptions).  Probably easiest if we add a new method `.render_exc_value(stack)` cribbing from [this part of the backport](https://github.com/agronholm/exceptiongroup/blob/e0575e8800cade057b5d1541b782b15a4b8670dc/src/exceptiongroup/_formatting.py#L335-L338)?

Reply:
Also wanted to do this, but again wanted to get initial feedback first as this will depend on how the maintainer was possibly envisioning. If I have the go ahead for how it should be achieved, I can make the changes. Right now we're stuck waiting for feedback.

Reply:
My intuition as a maintainer is that fewer rounds of review makes everything easier - if I can just say "thanks, merging" that's likely to happen _sooner_ than if we iterate.  I understand not wanting to do more work which might not be merged though. 🙂